docs/.vitepress/theme/custom.css

```
@@ -1536,282 +1628,428 @@ div[class*="language-"] .copy:hover {
+  text-decoration: underline;
+  text-decoration-color: var(--vp-c-divider);
+  text-underline-offset: 3px;
 }
 
-.recipe-tab-input {
-  position: absolute;
-  opacity: 0;
-  pointer-events: none;
+.landing-page .pantry-backends a:hover {
+  color: var(--vp-c-brand-1);
+  text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)